tests/data/ib_pset_problem_attempted_v1.json

```diff
@@ -0,0 +1,92 @@
+{
+    "data":[
+        {
+            "user_uuid": "c6a4a01e-b112-475f-82ff-b051ab1d44e1",
+            "course_id": 2,
+            "impression_id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+            "source_uri": "http://localhost:8000",
+            "timestamp": 0,
+            "eventname": "ib_pset_problem_attempted_v1",
+            "content_id": "3fa85f64-5717-4562-b3fc-2c963f66afa7",
+            "variant": "main",
+            "pset_content_id": "5dac90ce-fdbc-4ea9-bfe3-fccd13fa9d12",
+            "pset_problem_content_id": "743cb23f-527b-44b8-ac3b-39f88f8885b0",
+            "problem_type": "input",
+            "response": "some user input string",
```

Comment:
This was my oversight when writing the issue, but these `response` fields will currently look something like:

```json
"response": {
    "string": "some user input string",
    "array": null
}
```

Or, in the case below:

```json
"response": { 
    "string": null,
    "array": [
        "selected input 1",
        "selected input 2"
    ]
}
```

Currently the processor in `raise-data` just passes things through blindly so as not to be event type specific in it's logic. We could revisit that overall approach if we think it makes more sense to not have to deal with that at the last stage of the data flow :man_shrugging: .  This might be worth a live discussion to talk through tradeoffs, etc.